File: SFA-API/Domain/Route.cs
using System.Collections.Generic;

namespace Domain
{
    public class Route
    {
        public int Id { get; set; }
        public int AreaId { get; set; }
        public int StoreCount { get; set; }
        public string Name { get; set; }
        public string RouteCode { get; set; }
        public decimal StartLatitude { get; set; }
        public decimal StartLongitude { get; set; }
        public decimal EndLatitude { get; set; }
        public decimal EndLongitude { get; set; }
        public string Comment { get; set; }
        public virtual Area Area { get; set; }
        public virtual ICollection<Shop> Shops { get; set; }
        public virtual ICollection<Store> Stores { get; set; }
    }
}

File: SFA-Client/src/app/_models/route.ts
export class Route {
  id: number;
  districtId: number;
  provinceId: number;
  areaId: number;
  storeCount: number;
  name: string;
  routeCode: string;
  startLatitude: string;
  startLongitude: string;
  endLatitude: string;
  endLongitude: string;
  comment: string;

  constructor(route) {
    this.id = route.id;
    this.districtId = route.districtId;
    this.provinceId = route.provinceId;
    this.areaId = route.areaId;
    this.storeCount = route.storeCount;
    this.name = route.name;
    this.routeCode = route.routeCode;
    this.startLatitude = route.startLatitude;
    this.startLongitude = route.startLongitude;
    this.endLatitude = route.endLatitude;
    this.endLongitude = route.endLongitude;
    this.comment = route.comment;
  }
}
